src/main/java/domain/Lotto.java

```diff
@@ -0,0 +1,55 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Lotto {
+    private static final int MIN_LOTTO_NUMBER = 1;
+    private static final int MAX_LOTTO_NUMBER = 45;
+    private static final int LOTTO_START = 0;
+    private static final int LOTTO_END = 6;
+    private static final String DELIMITER = ", ";
+    private static final String OPEN_BRACKETS = "[";
+    private static final String CLOSE_BRACKETS = "]";
+    private final List<Integer> numbers;
```

Comment:
상수와 필드 사이 빈줄을 추가하면 가독성이 더 좋습니다.

```suggestion
    private static final int MIN_LOTTO_NUMBER = 1;
    private static final int MAX_LOTTO_NUMBER = 45;
    private static final int LOTTO_START = 0;
    private static final int LOTTO_END = 6;
    private static final String DELIMITER = ", ";
    private static final String OPEN_BRACKETS = "[";
    private static final String CLOSE_BRACKETS = "]";
    
    private final List<Integer> numbers;
```